test/e2e/status_test.go

```diff
@@ -0,0 +1,187 @@
+package e2e
```

Comment:
Should we have these tests in a separate file called `status_test.go`, since we need to test both the failure and successful scenarios in creation of nodes.